src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  closePost: (value: null) => void;
+  post: Todo | null;
+};
+
+export const TodoModal: React.FC<Props> = ({ closePost, post }) => {
+  const [selectUser, setSelectUser] = useState<User | null>(null);
```

Comment:
```suggestion
  const [selectedUser, setSelectedUser] = useState<User | null>(null);
```
selectUser sounds like a function